frontend/src/main/pages/HomePage.js

```diff
@@ -49,14 +49,6 @@ export default function HomePage() {
     }, [currentUser]
   );
 
-  useEffect(
-    () => {
-      if (commonsFromBackend) {
-        setCommons(commonsFromBackend);
-      }
-    }, [commonsFromBackend]
-  );
-
```

Comment:
So we're only fetching the commons when the component renders? I feel like we should be _polling_ the backend. If it fails, we have no cache from React's state. I don't understand React, so feel free to ignore.

Reply:
react-query handles caching and when requests should be performed. Unless it's a mutation, you should never have to imperatively make requests.